File: source/Nasty Hacks/nastyhacks.cpp
#include <bits/stdc++.h>

using namespace std;

int main()
{
    int n, r, e, c;
    cin >> n;
    for (int i=0; i<n; i++) {
        cin >> r >> e >> c;
        if (r+c < e) {
            cout << "advertise" << "\n";
        } else if (r+c > e) {
            cout << "do not advertise" << "\n";
        } else {
            cout << "does not matter" << "\n";
        }
    }

    return 0;
}

File: source/Nasty Hacks/nastyhacks.java
import java.util.Scanner;

public class nastyhacks {
    public static void main(String[] args) {
        Scanner in = new Scanner (System.in);
        int n = in.nextInt();
        for (int i = 0; i < n; i++) {
            int a = in.nextInt();
            int b = in.nextInt();
            int c = in.nextInt();
            if(a+c < b){
                System.out.println("advertise");
            }else if (a+c>b){
                System.out.println("do not advertise");
            }else{
                System.out.println("does not matter");
            }
            
        }
        
    }
}
